assets/demo-todo-app.spec.js

```diff
@@ -318,7 +318,7 @@ test.describe('Routing', () => {
   test('should allow me to display active items', async ({ page }) => {
     await page.locator('.todo-list li .toggle').nth(1).check();
     await checkNumberOfCompletedTodosInLocalStorage(page, 1);
-    await page.locator('.filters >> text=Active').click();
+    await page.locator('.filters').getByText('Active').click();
```

Comment:
```suggestion
    await page.getByRole('link', { name: 'Active' }).getByText('Active').click();
```

lets try to use such selectors for all the other ones. That's what codegen generates as well.